apps/ui/src/components/FormSpaceExecutionStrategies.vue

```diff
@@ -0,0 +1,144 @@
+  ],
+  queryFn: async () => {
+    const network = getNetwork(props.space.network);
+    const provider = getProvider(Number(network.chainId));
+
+    const avararExecutionStrategies = props.executionStrategies.filter(
+      strategy => strategy.type === 'SimpleQuorumAvatar'
+    );
+
+    const results: [string, boolean][] = [];
+
+    for (const strategy of avararExecutionStrategies) {
+      const treasury = treasuryMap.value[strategy.address];
+      if (!treasury) continue;
+
+      const safeContract = new Contract(treasury, SAFE_ABI, provider);
+
+      const isEnabled: boolean = await safeContract.isModuleEnabled(
+        props.space.id
+      );
+
+      results.push([strategy.address, isEnabled]);
+    }
+
+    return Object.fromEntries(results);
+  }
+});
+
+function getZodiacAppUrl(strategyAddress: string) {
+  const treasuryAddress = treasuryMap.value[strategyAddress];
+
+  return `https://app.safe.global/apps/open?safe=${props.space.network}:${treasuryAddress}&appUrl=https%3A%2F%2Fzodiac.gnosisguild.org%2F`;
+}
+</script>
+
+<template>
+  <div>
+    <UiLoading v-if="isPending" />
+    <UiStateWarning v-else-if="isError">
+      Error while loading execution strategies.
+    </UiStateWarning>
+    <div
+      v-for="executionStrategy in executionStrategies"
+      v-else
+      :key="executionStrategy.address"
+      class="flex justify-between items-center first-of-type:rounded-t-lg last-of-type:rounded-b-lg first-of-type:border-t border-b border-x px-4 py-3 text-skin-link"
+    >
+      <div class="flex items-center">
+        <UiBadgeNetwork
+          :chain-id="executionStrategy.chainId"
+          class="mr-3 hidden sm:block"
+        >
+          <UiStamp
+            :id="executionStrategy.address"
+            type="avatar"
+            :size="32"
+            class="rounded-md"
+          />
+        </UiBadgeNetwork>
+        <div class="flex-1 leading-[22px]">
+          <div class="flex items-center space-x-2">
+            <h4
+              class="text-skin-link"
+              v-text="
+                executionStrategy.name || shorten(executionStrategy.address)
+              "
+            />
+            <a
+              v-if="executionStrategy.type === 'SimpleQuorumAvatar'"
+              href="#"
+              target="_blank"
+              class="inline-block"
+            >
+              <IH-question-mark-circle />
+            </a>
+          </div>
+          <UiAddress
+            class="text-skin-text text-[17px]"
+            :address="executionStrategy.address"
+          />
+        </div>
+      </div>
+      <div class="flex gap-3">
+        <template
+          v-if="
+            moduleEnabled && executionStrategy.type === 'SimpleQuorumAvatar'
+          "
+        >
+          <UiButton
+            v-if="moduleEnabled[executionStrategy.address] === false"
+            :to="getZodiacAppUrl(executionStrategy.address)"
+          >
+            Enable module
```

Comment:
I would rename to just Enable, it will save some horizontal space for mobile and match oSnap button (which is also enabling a Safe module)
```suggestion
            Enable
```